### ISSUE TITLE
choosing a radio button by alt text

### ISSUE THREAD

**Issue by ashanbrown** [teamcapybara/capybara#1356]
I have a radio button where the label is an image and it appears to me that capybara cannot select a radio button (or presumably checkbox either) using just the alt text of the image.  It can, however, perform click_link on an image based on alt text.  I realize it becomes sort of murky when you combine text and images with alt text  -- I'm not sure what happens for links in this case -- but does this sound like something capybara should support?  If so, I might investigate generating a pull request for this functionality.  It might, however, require deciding how text is matched when you have images and text combined (concatenation seems like the most obvious choice, although if the image is in front and you want to select by the visible label text, then simple concatenation would not work).  Thanks,

Andrew
